tests/phpunit/includes/abstract-testcase.php

```diff
@@ -1864,4 +1864,17 @@ private function retry_on_error( callable $callback, $url, $args ) {
 
 		return $result;
 	}
+
+	/**
+	 * Clean up the unzip destination.
+	 *
+	 * @param string $unzip_destination Unzip destination.
+	 * @return void
```

Comment:
WP treats `@return void` as implicit and doesn't document them, a since tag is also required.

```suggestion
	 * Clean up the unzip destination.
	 *
	 * @since 6.9.0
	 *
	 * @param string $unzip_destination Unzip destination.
```